client/boot/common.js

```diff
@@ -264,10 +282,23 @@ function setupErrorLogger( reduxStore ) {
 		errorLogger.saveExtraData( { lastTracksEvent } )
 	);
 
+	let prevPath;
```

Comment:
Could you elaborate on what this variable is used for?

Reply:
Yes, so sentry navigation breadcrumbs need to have a properties for both the page you navigate _from_ and the page you navigate _to_. Keeping track of the `prevPath` means we can maintain the "from" data, and then the "to" data is just the current page in the page navigation handler below.

We just update this variable after sending the breadcrumb to sentry with the new "from" page, so that it's available for the next event.

Reply:
Ah, I see. I was initially confused because I thought this script ran in the SSR context, which would mean this variable would be shared across requests. Now I see it's the *client*-side route handler, and now it makes sense!